README.md

```diff
@@ -79,6 +80,7 @@ Adding the following to your workflow will create a new Sentry release and tell
       with:
         environment: 'production'
         sourcemaps: './lib'
+        dist: ${{ github.sha }}
```

Comment:
Does the `github.sha` not get figured out automatically? I thought the usage of `dist` is more for when we do not want `github.sha` to be used.

I do not know the code enough but that's what I remember.

Reply:
oooh this is interesting... I wonder if this is a bug on its own then.

When using the github action as it is currently, `dist` is not set at all and shows as "null" in the Sentry UI, which causes source maps not to match up at all.

Without this change, this is what I see in the Sentry UI:
<img width="1191" alt="Screen Shot 2023-02-24 at 12 10 23 PM" src="https://user-images.githubusercontent.com/153077/221052071-734df804-514b-4eb3-a9ea-4298e2a26631.png">

Note the "none" on the artifacts. This should be the `dist` and sourcemaps won't work correctly without the `dist` value matching on both sourcemaps and the code/application at runtime (via eg. `Sentry.init({ dist })` in JS`)

Reply:
If it's reasonable for `dist` to default to the github SHA, i could likely update the PR to do that. There isn't much in the way of docs about what `dist` should be, other than a suggestion it might be your build number (?) but that doesnt seem correct for use in GH workflows -- SHA seems more reasonable.

Reply:
added screenshots to the main issue description too for before/after